app/controllers/api_controller.rb

```diff
@@ -57,6 +57,8 @@ def info_for_paper_trail
 private
 
   def set_sentry_transaction_id
+    return unless Sentry.initialized?
```

Comment:
Is this to resolve an issue with the initialisation of Sentry?

Reply:
Yeah, this is unrelated to the PER author stuff, it just means that we can run the API locally without setting the sentry env vars now